### PR TITLE
Add business id to cronjob chart

### PR DESCRIPTION
**Pull request by abdullah-sl**

### CODE REVIEW

File: cronjob/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Helm chart with simple cronjob template
 name: cronjob
-version: 0.9.0
+version: 0.10.0
```
Comment (Lokman928) [inline cronjob/Chart.yaml:4]: Remember update the version before merge

Comment (ThomasKwan-shopline) [inline cronjob/Chart.yaml:4]: remember update version before merge

File: cronjob/templates/cronjob.yaml [hunk 2]
```diff
@@ -10,6 +10,8 @@ kind: CronJob
 metadata:
   name: {{ .Values.name }}
   namespace: {{ .Release.Namespace }}
+  labels:
+    businessid: {{ required "businessid must be provided" .Values.businessid | quote }}
```
Comment (ThomasKwan-shopline) [inline cronjob/templates/cronjob.yaml:14]: NIT: why add required here, should add required in all businessid field?

Comment (abdullah-sl) [reply cronjob/templates/cronjob.yaml:14]: This is the entry point, so I believe checking here would be enough.

**Review by thomaslishopline — APPROVED** [review]

**Review by ThomasKwan-shopline — APPROVED** [review]

**Review by ThomasKwan-shopline — DISMISSED** [review]

**Review by thomaslishopline — DISMISSED** [review]

**Review by neal-sl — APPROVED** [review]

**Review by ThomasKwan-shopline — APPROVED** [review]

**Review by thomaslishopline — APPROVED** [review]

**Pull request merged**